task/06-conditions-n-loops-tasks.js

```diff
@@ -82,7 +107,12 @@ function getSumBetweenNumbers(n1, n2) {
  *   10,10,10 =>  true
  */
 function isTriangle(a,b,c) {
-    throw new Error('Not implemented');
+    if((a + b > c) && (b + c > a) && (a + c > b)){
```

Comment:
use `return <bolean expression>` instead of `if (<bolean expression>) return true else return false`